source/renderer/app/components/staking/delegation-center/DelegationCenter.js

```diff
@@ -1,33 +1,39 @@
 // @flow
 import React, { Component, Fragment } from 'react';
 import { observer } from 'mobx-react';
-import BigNumber from 'bignumber.js';
 import DelegationCenterHeader from './DelegationCenterHeader';
 import DelegationCenterBody from './DelegationCenterBody';
 import Wallet from '../../../domains/Wallet';
+import type { TipInfo } from '../../../api/network/types';
 
 type Props = {
-  adaValue: BigNumber,
-  percentage: number,
   wallets: Array<Wallet>,
   onDelegate: Function,
   numberOfStakePools: number,
+  startDateTime: string,
```

Comment:
@DeeJayElly please change to `nextEpoch` and pass the next epoch number and start date-time through it...

Reply:
Fixed